evaluators/rank/f1ScoreEvaluator/__init__.py

```diff
@@ -0,0 +1,39 @@
+from jina.executors.evaluators.rank import BaseRankingEvaluator
+
+class f1ScoreEvaluator(BaseRankingEvaluator):
+    """
+    :class:`f1ScoreEvaluator` Gives the f1 score of a result given the groundtruth..
+    """
+
+    def __init__(self, eval_at ,*args, **kwargs):
```

Comment:
we need a default value to pass the tests

Reply:
it may need to be moved to the base class but put it here for now

Reply:
I think 1 would make a better default

Reply:
probably,but can you explain ?

Reply:
Assuming that it should not be defined here, we may address this in another PR. We need to provide a k to do an eval at. 0 seems like the last eval at point someone would use because it does not involve any evaluation actually. Don't you think so?

Reply:
you're right